oozie/src/main/java/org/apache/falcon/oozie/feed/HCatReplicationWorkflowBuilder.java

```diff
@@ -72,6 +84,16 @@ public HCatReplicationWorkflowBuilder(Feed entity) {
             OozieUtils.unMarshalHiveAction(export);
         org.apache.falcon.oozie.hive.ACTION hiveExportAction = exportActionJaxbElement.getValue();
         addHDFSServersConfig(hiveExportAction, src, target);
+        Properties hiveConf = HiveUtil.getHiveCredentials(src);
```

Comment:
This looks similar to the code you added at line 66-74. Better to organize them as one function that propagates hive credential to an action.